partcad/src/partcad/__init__.py

```
@@ -84,17 +91,77 @@ def before_send(event, hint):
+    # # "null - Replaced because of SDK configuration"
+    # sentry_sdk.set_user({"ip_address": "{{auto}}"})
+
+    sentry_sdk.set_tag("utm.source", "PartCAD")
+    sentry_sdk.set_tag("utm.medium", "Sentry")
+    sentry_sdk.set_tag("utm.campaign", "Open Source")
+    sentry_sdk.set_tag("utm.content", "API")
+
+    sentry_sdk.set_tag("env.remote_containers", os.environ.get("REMOTE_CONTAINERS", "false").lower() == "true")
+
+    hostname = socket.getfqdn()
+    hostname_md5 = hashlib.md5(hostname.encode()).hexdigest()
```

Ruff (0.8.2)

Probable use of insecure hash functions in `hashlib`: `md5`

(S324)